data/brands/amenity/bicycle_rental.json

```
@@ -4122,6 +4122,23 @@
         "name": "高雄捷運股份有限公司",
         "operator": "高雄捷運股份有限公司"
       }
+    },
+    {
+      "displayName": "杭州公共自行车",
+      "locationSet": {"include": ["cn"]},
```

```suggestion
      "locationSet": {"include": ["cn"],"exclude":["hk","mo"]},
```
Because Hangzhou is in Mainland of China, so I exclude HK and MO, this will help mapper fliter invaild preset.

---

OK thank you